test/runtests.jl

```diff
@@ -1,5 +1,13 @@
 using CoordinateTransformations
-using Base.Test
+using BaseTestNext
+using FixedSizeArrays
+import ForwardDiff.GradientNumber
+Base.show(io::IO, g::GradientNumber) = print(io, "GradientNumber($(g.value), $(g.partials.data))") # Output was incomprehensible!
 
-# write your own tests here
-@test 1 == 1
+@testset "CoordinateTransformations" begin
+
+    include("core.jl")
+    include("coordinatesystems.jl")
+    include("commontransformations.jl")
```

Comment:
Impressive test effort :)  The derivative testing vs `GradientNumber` is pretty neat.